crosslink/internal/crosslink.go

```diff
@@ -19,46 +19,68 @@ import (
 	"path/filepath"
 	"strings"
 
+	"go.uber.org/zap"
 	"golang.org/x/mod/modfile"
 )
 
-func Crosslink(rc runConfig) {
+func Crosslink(rc RunConfig) {
 	var err error
 
 	rootModulePath, err := identifyRootModule(rc.RootPath)
 	if err != nil {
-		panic(fmt.Sprintf("failed to identify root module: %v", err))
+		rc.Logger.Sugar().Panic("Failed to identify root Module",
```

Comment:
```suggestion
		rc.Logger.Panic("Failed to identify root Module",
```

I don't think the sugared logger is needed for this pattern.